core/includes/errors.inc

```diff
@@ -25,7 +25,6 @@ function backdrop_error_levels() {
     E_USER_ERROR => array('User error', WATCHDOG_ERROR),
     E_USER_WARNING => array('User warning', WATCHDOG_WARNING),
     E_USER_NOTICE => array('User notice', WATCHDOG_NOTICE),
-    E_STRICT => array('Strict warning', WATCHDOG_DEBUG),
```

Comment:
My suspicion is, that we still need E_STRICT for PHP versions before 8.4.
It's mentioned in every php.ini for every version other than 8.4.
We could just remove it for 8.4, leave it in place for all previous versions.

Otherwise something might slip through...

Reply:
Thanks, I added a version check and restored E_STRICT to the list by appending it to the end of the array if on versions older than 8.4.